release-process.md

```diff
@@ -0,0 +1,52 @@
+<!--
+Copyright 2021 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# The Aquarius Release Process
+
+Please note that this upgrade does not affect the deployed versions of Aquarius, i.e. http://aquarius.oceanprotocol.com. Contact the Ocean Protocol team for such a deployment.
+
+## Step 0: Update documentation
+
+- Go to https://github.com/oceanprotocol/readthedocs, and follow the steps
+- This will update what's shown in https://docs.oceanprotocol.com/references/read-the-docs/aquarius/.
+
+This doesn't actually affect the pip release of the following steps. And if you've just updated READMEs, you can stop after this step if you like.
```

Comment:
Mentions "pip" release but aquarius no longer does PyPi releases. Rather, it's released as a docker image.